src/libraries/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionary.Generic.Tests.cs

```diff
@@ -85,10 +85,9 @@ public abstract class ConcurrentDictionary_Generic_Tests<TKey, TValue> : IDictio
     {
         #region IDictionary<TKey, TValue Helper Methods
 
-        protected override IDictionary<TKey, TValue> GenericIDictionaryFactory()
-        {
-            return new ConcurrentDictionary<TKey, TValue>();
-        }
+        protected override IDictionary<TKey, TValue> GenericIDictionaryFactory() =>new ConcurrentDictionary<TKey, TValue>();
+
+        protected override IDictionary<TKey, TValue> GenericIDictionaryFactory(IEqualityComparer<TKey> comparer) => new Dictionary<TKey, TValue>(comparer);
```

Comment:
Unless I'm missing something, shouldn't this be creating a ConcurrentDictionary?

Reply:
Ugh, you're not missing anything :-)